nebula-logger/extra-tests/tests/Logger_Tests_Network.cls

```diff
@@ -3,15 +3,11 @@
 // See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    //
 //------------------------------------------------------------------------------------------------//
 
-/**
- * @group Extra Tests
- * @description Additional integration tests for orgs with Experience Sites (Communities) enabled
- */
 @SuppressWarnings('PMD.ApexDoc, PMD.CyclomaticComplexity, PMD.ExcessiveParameterList, PMD.MethodNamingConventions, PMD.NcssMethodCount')
 @IsTest(IsParallel=false)
-private class Logger_Tests_ExperienceSite {
-    private static final Boolean IS_EXPERIENCE_CLOUD_ENABLED = Schema.getGlobalDescribe().containsKey('Network');
-    private static final String GUEST_USER_PROFILE_NAME = 'Logger Test Site Profile';
+private class Logger_Tests_Network {
+    private static final Boolean IS_EXPERIENCE_CLOUD_ENABLED = Type.forName('System.Network') != null;
```

Comment:
I thought it was `NetworkMember` that provided the crucial piece of info?

Reply:
....whoops! You're right, thanks for catching this! `NetworkMember` is the useful one to check - I have this check duplicated in a few classes & missed updating this one. I'll have to eventually add a central location for this check so I can cut out the duplication.

Reply:
cool!